### ISSUE TITLE
Console.debug is missing in NodeVM.

### ISSUE THREAD

**Issue by momocow** [patriksimek/vm2#152]
This is my NodeVM.
```js
const vm = new NodeVM({
          console: 'redirect',
          sandbox: {
            setTimeout,
            setInterval,
            setImmediate
          },
          require: {
            external: true
          }
        })
          .on('console.log', (msg, ...args) => {
            console.log('!!! ' + msg, ...args)
          })
          .on('console.info', (msg, ...args) => {
            console.log('!!! ' + msg, ...args)
          })
          .on('console.warn', (msg, ...args) => {
            console.log('!!! ' + msg, ...args)
          })
          .on('console.error', (msg, ...args) => {
            console.log('!!! ' + msg, ...args)
          })
const {default: Test} = vm.require("abs-path-to-module")
const instance= new Test()
```

And my codes to run in the VM.
```js
// path: "abs-path-to-module/index.js"

// console.debug('[debug] outside')
console.info('[info] outside')
console.log('[log] outside')
console.warn('[warn] outside')
console.error('[error] outside')

export default class LoggerTest {
  constructor () {
    // console.debug('[debug] outside')
    console.info('[info] outside')
    console.log('[log] outside')
    console.warn('[warn] outside')
    console.error('[error] outside')
  }
}

```

Console output is expected have prefix "!!! ".
```
[info] outside
[log] outside
[warn] outside
[error] outside
[info] outside
[log] outside
[warn] outside
[error] outside
```

-------------

Also, I was facing a TypeError thrown from the codes run in a NodeVM.
```
TypeError: console.debug is not a function
```

Quoted from #54
> You can listen for those events:
> 
> - `console.log`
> - `console.info`
> - `console.warn`
> - `console.error`
> - `console.dir`
> - `console.trace`

I found that `console.debug` is missing.
I consider it is a bug that a NodeVM should follow the NodeJS Console API.
https://nodejs.org/api/console.html#console_console_debug_data_args

Thanks if there's any help!

**Comment by momocow** [comment]
I have some more tests for this.

I tested the redirection manually in the REPL and it worked.
```js
const vm = new (require('vm2').NodeVM)({console:'redirect'})
vm.on('console.log', console.log.bind(console, '!!!'))
vm.run('console.log("test")')
// Print: "!!! test"
// Return: {}
```

Since my project is a electron project, I use [electron-webpack](https://github.com/electron-userland/electron-webpack) to package the codes. Maybe it has some compatibility issues with vm2 I guess. I'll look more into it to find out why the redirection failed.

It seems that it is not a vm2 issue so far.

---------


However, `console.debug` is still the problem. The error it throws is not a VMError but a TypeError (not knowing if it is expected or not).
```js
// to make sure that console.debug is in the host environment
console.debug('test')
// Print: "test"
// Return: undefined

vm.run('console.debug("test")')
```
```
TypeError: console.debug is not a function
    at Object.<anonymous> (vm.js:1:71)
    at NodeVM.run (<my-project>\node_modules\vm2\lib\main.js:419:27)
    at repl:1:4
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at REPLServer.defaultEval (repl.js:240:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
    at REPLServer.onLine (repl.js:468:10)
    at emitOne (events.js:121:20)
    at REPLServer.emit (events.js:211:7)
```

**Comment by momocow** [comment]
I've found the problem on the console redirection. I have filed it into another issue #153 .

**Comment by patriksimek** [comment]
To be fixed in 3.6.10. Thank you for reporting the issue.
